docs/getting-started.rst

```diff
@@ -4,101 +4,244 @@
 
-If your protein is not recognised by Pfam, you will need to paste the protein sequence into the search page. We will search your sequence against our HMMs and instantly display the matches for you.
+1. Click the **Search + By Sequence** in the InterPro website menu. This opens the **Search InterPro** page. 
 
-Batch search
-------------
+.. figure:: images/searchbyseq.png
+    :alt: Selecting **Search + By Sequence** in the InterPro website menu
+    :width: 700
+    :align: left
+
+    Selecting **Search + By Sequence** in the InterPro website menu.
+
+2. Provide the FASTA formatted sequence(s) (protein or DNA) of interest by pasting them into the test dialogue box or by importing them from a file
+
+.. figure:: images/fasta.png
+    :alt: Selecting **Search + By Sequence** in the InterPro website menu
+    :width: 700
+    :align: left
+
+    Example of protein sequence in FASTA format in the **Enter your sequence box**.
+
+3. Expand the **Advanced options**, click on **Unselect all** protein sequence applications and select Pfam
+
+.. figure:: images/advanced_pf.png
+    :alt: Selecting **Pfam** in the **Advanced options** drop-down
+    :width: 700
+    :align: left
+
+    Select only **Pfam** to search your sequence(s) against this database.
 
-If you have a large number of sequences to search (up to several thousand), you can use our batch upload facility. This allows you to upload a file of your sequences in FASTA format, and we will run them against our HMMs and email the results back to you, usually within 48 hours. We request that you put a maximum of 5000 sequences in each file.
+
+4. Click the Search button
+
+While the sequence search is running, you can continue to navigate through the website, other browser tabs or applications and will get a pop-up notification when the job has been completed (this requires the browser notifications to be allowed).
+
+The results of the submitted job are accessible by selecting **Results + Your InterProScan Searches** in the InterPro website menu. 
+
+.. figure:: images/results.png
+    :alt: Selecting **Results + Your InterProScan Searches**
+    :width: 700
+    :align: left
+
+    Select **Results + Your InterProScan Searches** in the InterPro website menu. 
+
+All Pfam signatures - and the InterPro entries where they are integrated - are displayed in the protein sequence viewer. The Pfam signatures and InterPro entries are grouped by type (family, domain, repeat, site). The coloured bars indicate the location of signature matches on the protein sequence. Each matched InterPro entry is displayed on a separate line, with the signatures integrated in it displayed below.
+
```

Comment:
add image example of protein sequence viewer

Reply:
The results from the example sequence are quite simple, maybe it would be a good idea to look here for an example with several sections in the protein viewer (family, domain,unintegrated..), that search lead me to wonder if we should explain somewhere what PfamN is? Maybe just a very brief text in the 'Pfam entries creation and annotation' section with a link to the corresponding blog post?

Reply:
Yes we should definitely add a section about Pfam-N. You can mention it in this section, no need to develop too much as the blog post explains in details.